### PR TITLE
fix: [FFM-6745]: Fixing cut off text in Delete Target Group dialog

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

Previously the text did not wrap properly:

image

This was happening across the Target Group delete dialogues.

#### Screenshots

Now we split the sentences so that the UX is better. This has been ran by Ksenia.

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by knagurski — APPROVED** [review]
Nice and simple and good job updating those older `dangerouslySetInnerHTML` usages as well 👍🏻

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -308,7 +308,7 @@ segments:
   usingSegmentWithCount: Flags using this target group ({{ count }})
   modalTitle: Create a Target Group
   delete:
-    message: 'Are you sure you want to delete target group <strong>{{segmentName}}</strong>? This action cannot be undone.'
+    message: 'Are you sure you want to delete target group <strong>{{segmentName}}</strong>? <br/>This action cannot be undone.'
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:311]: should we capitalise Target Group while we're here?

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -308,7 +308,8 @@ segments:
   usingSegmentWithCount: Flags using this target group ({{ count }})
   modalTitle: Create a Target Group
   delete:
-    message: 'Are you sure you want to delete target group <strong>{{segmentName}}</strong>? This action cannot be undone.'
+    message: 'Are you sure you want to delete target group <strong>{{segmentName}}</strong>?'
```
[the remark this answers is not in the document]
Comment (knagurski) [reply src/modules/75-cf/strings/strings.en.yaml]: lol, _technically_ we probably should have changed the CSS to tell it not the break things (it probably has `break-word: break-all` set somewhere for some reason), but a line break makes a ton of sense for this specific case.

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: I hate this but you are right lol have capitalised.

**Review by crystal-fong — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
🔥🔥

**Pull request merged**